test/oban/worker_test.exs

```diff
@@ -95,6 +95,16 @@ defmodule Oban.WorkerTest do
     end
   end
 
+  describe "from_string/1" do
+    test "finds workers that are not loaded" do
+      :code.delete(Oban.Integration.Worker)
```

Comment:
I believe this may cause race conditions due to async tests. To be safe, this module should have `async: false`.